Main.java

```diff
@@ -3,74 +3,145 @@
 import java.io.*;
 import java.nio.file.Files;
 import java.nio.file.Paths;
+import java.sql.*;
 import java.util.StringTokenizer;
 
+import static com.company.Main.ERROR_MESSAGE;
+import static com.company.Main.statement;
+
 public class Main {
 
-    final static String FILE_TYPE_TXT = ".txt";
-    final static String FILE_TYPE_JSON = ".json";
-    final static String REMOVE_SPECIAL_CHAR = "\\p{Punct}";
-    final static String SINGLE_SPACE = "";
+    static final String FILE_TYPE_TXT = ".txt";
+    static final String FILE_TYPE_JSON = ".json";
+    static final String REMOVE_SPECIAL_CHAR = "[^a-zA-Z0-9 -]";
+    static final String SINGLE_SPACE = "";
+    static String ERROR_MESSAGE;

```

Comment:
ALL CAPS is only used for constants. You actually do not need to have ERROR_MESSAGE as a static field here, it can just be a local variable